internal/services/devtestlabs/dev_test_lab_resource.go

```diff
@@ -100,6 +101,21 @@ func resourceDevTestLab() *pluginsdk.Resource {
 			},
 		},
 	}
+
+	if !features.FourPointOhBeta() {
```

Comment:
we'll need to wrap the code in the create/read/update funtions to handle this property not existing anymore?

Reply:
Updated. Thanks.